vscode-trace-extension/package.json

```diff
@@ -89,6 +89,11 @@
         "title": "Open Trace",
         "icon": "$(new-folder)"
       },
+      {
+        "command": "trace-explorer.refreshContext",
+        "title": "Refresh",
```

Comment:
![image](https://github.com/eclipse-cdt-cloud/vscode-trace-extension/assets/25749063/709a923a-6870-435e-a708-6e8264d067c8)

The name should be more descriptive. Something like "Refresh Trace Server"?

Reply:
I think it should be `Refresh Trace Viewer` or `Refresh Trace Explorer` because it's the front-end view that is being refreshed using the back-end sever.

Reply:
addressed